packages/runtime-core/__tests__/apiCreateApp.spec.ts

```diff
@@ -93,6 +93,14 @@ describe('api: createApp', () => {
     app.provide('foo', 1)
     app.provide('bar', 2)
 
+    const foo = app.inject('foo')
+    const bar = app.inject('bar')
```

Comment:
Perhaps you should take the injection keys to constants, so that you do not make a mistake with the key and it was possible to change the key in only one place. Do you think this is worth doing?